ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.c

```diff
@@ -195,6 +195,53 @@ static void gst_mtl_st20p_rx_class_init(Gst_Mtl_St20p_RxClass* klass) {
                           "v210", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 }
 
+typedef struct st20_rx_external_data {
+  GstBuffer* buf;
+  GstMapInfo dest_info;
+} st20_rx_external_data_t;
+
+static int st20p_rx_query_ext_frame(void* priv, struct st_ext_frame* ext_frame,
```

Comment:
functions should have a definition on the top 
even when they are static to follow the convection

Reply:
done